crypto/ec/ec_oct.c

```diff
@@ -74,6 +74,10 @@ size_t EC_POINT_point2oct(const EC_GROUP *group, const EC_POINT *point,
                           point_conversion_form_t form, unsigned char *buf,
                           size_t len, BN_CTX *ctx)
 {
+    if ( point == NULL){
```

Comment:
Space between the `)` and the `{` please.

Reply:
Done. Plz review again!